designs/wahid/src/frontfacing.js

```diff
@@ -29,19 +27,42 @@ export default (part) => {
     paths.seam.line(points.closureBottom).line(points.dartHemLeft)
   }
   paths.seam.close().attr('class', 'fabric')
-
   if (complete) {
-    if (sa) paths.sa = paths.seam.offset(sa).attr('class', 'fabric sa')
+    // Grainline
+    if (options.hemStyle === 'classic') {
+      points.grainlineFromFrontFacing = new Point(points.hemTip.x, points.closureTop.y)
+      points.grainlineToFrontFacing = new Point(points.hemTip.x, points.hemTip.y).shift(90, 20)
+    }
+    if (options.hemStyle === 'rounded') {
+      points.grainlineFromFrontFacing = new Point(points.roundEnd.x, points.roundEnd.y).shift(
+        90,
+        200
+      )
+      points.grainlineToFrontFacing = new Point(points.roundEnd.x, points.roundEnd.y)
+    }
+    if (options.hemStyle === 'square') {
+      points.grainlineFromFrontFacing = new Point(points.cutonfoldTo.x, points.cutonfoldTo.y).shift(
+        90,
+        200
```

Comment:
Similarly, the front facing grainline extends past the pattern with the square hem.

![Screen Shot 2022-08-13 at 7 05 45 PM](https://user-images.githubusercontent.com/109869956/184519650-b992884b-af4b-4f12-88e1-68f5ef6ad726.png)